packages/react-components/src/components/MessageThread.tsx

```diff
@@ -849,7 +849,7 @@ export const MessageThread = (props: MessageThreadProps): JSX.Element => {
       if (onLoadPreviousChatMessages) {
         isLoadingChatMessagesRef.current = true;
         // Fetch message until scrollTop reach the threshold for fetching new message
-        while (!isAllChatMessagesLoadedRef.current && chatScrollDivRef.current.scrollTop <= 500) {
+        while (!isAllChatMessagesLoadedRef.current && chatScrollDivRef.current.scrollTop <= 0) {
```

Comment:
I think we'll still need some tolerance value - does reducing this to `50px` instead of `0` still work? I'll play around in storybook and see what the behavior is like with `0px` on windows

Reply:
50px is a hit or miss. In safari, the scrollbar can actually stick to the top < 50. 
From my testing, I've had the scrollbar stick as low as 19px from the top. (Hard to repro, but I was able to).

Reply:
At any value greater than 0 it can theoretically stick to the top given enough messages/ height of the browser window.  0 is the only value that guarantees no subsequent loading of previous messages. @JamesBurnside I was wondering why we need a tolerance value.

Reply:
My gut feeling was scrollTop sometimes doesn't go to 0 when scrolling (it can stick at like 1 or 5) but I cannot repro nor find any historical context to support that.
But maybe get @PorterNan to sign off on this as he added the original 200 check: https://github.com/Azure/communication-ui-library/pull/146/files#diff-a7cad3c035297ccc8b0624d8faffb541631f0c836f4c2bc3383bae7d76581f55
Then upped it to 500: https://github.com/Azure/communication-ui-library/pull/657/files#diff-ab8c8370a350df97f2fd3fb1e1ebaa994f438c4b8f0b569552d92ca1b70087e5
@PorterNan do you recall the reason for the 500? (especially since it was upped from 200?)